setup.py

```diff
@@ -135,7 +135,7 @@ def get_outputs(self):
     },
     test_suite = 'nose.collector',
     install_requires = [
-        'google-auth==1.4.2',
+        'google-auth==1.6.3',
```

Comment:
won't you run into this issue again if you pin the version here?

Reply:
Yes, but I wasn't sure if you waned to keep the pin. I can just change it to `>=`